build/optaplanner-build-parent/pom.xml

```diff
@@ -74,6 +74,8 @@
     <!-- Latest release to be used by api-compatibility-check to check backwards compatibility of the API. -->
     <revapi.oldKieVersion>7.39.0.Final</revapi.oldKieVersion>
     <revapi.newKieVersion>${project.version}</revapi.newKieVersion>
+    <!-- Runs only in the fullProfile. -->
+    <revapi.skip>true</revapi.skip>
```

Comment:
I believe revapi needs to run with a "mvn clean install" too, but not a "mvn clean install -Dquickly", because it's too often an issue when submitting a PR without a maven build.

Reply:
+1.